daemon/image_inspect.go

```diff
@@ -14,7 +14,7 @@ import (
 func (daemon *Daemon) LookupImage(name string) (*types.ImageInspect, error) {
 	img, err := daemon.GetImage(name)
 	if err != nil {
-		return nil, fmt.Errorf("No such image: %s", name)
+		return nil, fmt.Errorf("no such image: %s, cause: %v", name, err)
```

Comment:
no big preference - other than the missing `cause: ` phrase makes a little less hard to parse/read.